utils/validation/program.ts

```diff
@@ -0,0 +1,25 @@
+import { Program } from "models/Program";
+
+/**
+ * validateProgram takes programData of type Program and validates the data
+ * @param newProgramData - data corresponding to new program
+ * @returns true: validated, false: not validated
+ */
+export function validateProgram(newProgramData: Program) {
```

Comment:
Same comment as above, perhaps change the type to be more accurate. We can then change this to `validateProgramCreate` or `validateCreateProgram`.